jobApplicationTracker/Models/Interview.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace jobApplicationTrackerApi.Models
+{
+    public class Interview
```

Comment:
I do not see Navigation Properties. It's related to all models

Reply:
Thanks for pointing that out, Larry! You're absolutely right we need navigation properties to simplify loading related data and allow querying of relationships. I realize I forgot to inform Ivan about this earlier. Appreciate the feedback! 👍